pkg/scripts/bigchaindb-monit-config

```diff
@@ -93,7 +93,7 @@ case \$1 in
   start_bigchaindb)
 
     pushd \$4
-      nohup bigchaindb start >> \$3/bigchaindb.out.log 2>> \$3/bigchaindb.err.log &
+      nohup bigchaindb -l DEBUG start >> \$3/bigchaindb.out.log 2>> \$3/bigchaindb.err.log &
```

Comment:
Should the log level be a variable? if `bigchaindb-monit-config` is intended to run and monitor production system then user should be able to select the necessary/required log level.